options/locale/locale_en-US.ini

```diff
@@ -2591,7 +2591,7 @@ find_file.no_matching = No matching file found
 error.csv.too_large = Can't render this file because it is too large.
 error.csv.unexpected = Can't render this file because it contains an unexpected character in line %d and column %d.
 error.csv.invalid_field_count = Can't render this file because it has a wrong number of fields in line %d.
-error.broken_git_hook = Git hooks of this repository seem to be broken. Please follow the <a target="_blank" rel="noreferrer" href="https://docs.gitea.com/help/faq#push-hook--webhook-arent-running">documentation</a> to fix them, then push some commits to refresh the status.
+error.broken_git_hook = Git hooks of this repository seem to be broken. Please follow the <a target="_blank" rel="noreferrer" href="https://docs.gitea.com/next/help/faq#push-hook--webhook--actions-arent-running">documentation</a> to fix them, then push some commits to refresh the status.
```

Comment:
I still think it should be put into tmpl, because the link here would be unchangeable.

ps: I do not think it should point to "next". It should just point to the default latest stable.

Reply:
Sorry, "next" is a copy-paste typo.

What do you mean that "the link here would be unchangeable"?

Reply:
https://github.com/go-gitea/gitea/pull/29494#discussion_r1507592659

Reply:
I understand now. Let's continue the discussion at https://github.com/go-gitea/gitea/pull/29494#discussion_r1508457142.